pysat/_instrument.py

```diff
@@ -2632,18 +2585,24 @@ def load(self, yr=None, doy=None, end_yr=None, end_doy=None, date=None,
                 if (self.index[-1] == last_pad) & (not want_last_pad):
                     self.data = self[:-1]
 
-        # if self.pad is False, load single day
+        # If self.pad is False, load single day
         else:
             self.data, meta = self._load_data(date=self.date, fid=self._fid,
                                               inc=self.load_step)
             if not self.empty:
+                meta.accept_default_labels(self.meta)
                 self.meta = meta
 
+                # If only some metadata included, define the remaining variables
+                for var in self.variables:
+                    case_var = meta.var_case_name(var)
+                    if case_var not in self.meta.keys() \
+                       and case_var not in self.meta.keys_nD():
+                        self.meta[case_var] = {self.labels.name: var}
+
         # check if load routine actually returns meta
         if self.meta.data.empty:
-            self.meta[self.variables] = {self.labels.name: self.variables,
-                                         self.labels.units:
-                                         [''] * len(self.variables)}
+            self.meta[self.variables] = {self.labels.name: self.variables}
```

Comment:
The same could be said for this statement. I think I originally added it before unit tests. 

I'm not opposed to a soft-floor for standards to help users out but given how standards have steadily increased for pysat we may want to modify the approach.

Reply:
This was posted in a weird order. The `The same could be said for this statement` refers to the check being a unit test.

Reply:
This statement is useful, but your comments are a bit muddled and the clarification not as clear as I would like to be able to respond.  Perhaps we should have a chat about your comments?

Reply:
Sure.